### ISSUE TITLE
Are you Buddhist?

### ISSUE THREAD

**Issue by TangMonk** [alexamies/buddhist-dictionary#27]

**Comment by alexamies** [comment]
Yes and glad you discovered this repo. It drives the site https://ntireader.org. Hope you find it helpful either by using the repo or the site.

**Comment by TangMonk** [comment]
> Yes and glad you discovered this repo. It drives the site https://ntireader.org. Hope you find it helpful either by using the repo or the site.

I am also create an iOS app and list on App Store: https://apps.apple.com/en_US/app/%E4%BD%9B%E7%BB%8F%E5%A4%A7%E5%85%A8-%E6%B7%B1%E5%85%A5%E7%BB%8F%E8%97%8F/id1507655861

And also open source on Github: https://github.com/TangMonk/dufoj
